libraries/swagger/ConnectorAPI.json

```diff
@@ -158,7 +158,7 @@
             "Conversations"
           ],
           "summary": "CreateConversation",
-          "description": "Create a new Conversation.\r\n\r\nPOST to this method with a\r\n* Bot being the bot creating the conversation\r\n* IsGroup set to true if this is not a direct message (default is false)\r\n* Array containing the members to include in the conversation\r\n\r\nThe return value is a ResourceResponse which contains a conversation id which is suitable for use\r\nin the message payload and REST API uris.\r\n\r\nMost channels only support the semantics of bots initiating a direct message conversation.  An example of how to do that would be:\r\n\r\n```\r\nvar resource = await connector.conversations.CreateConversation(new ConversationParameters(){ Bot = bot, members = new ChannelAccount[] { new ChannelAccount(\"user1\") } );\r\nawait connect.Conversations.SendToConversationAsync(resource.Id, new Activity() ... ) ;\r\n\r\n```",
+          "description": "Create a new Conversation.\r\n\r\nPOST to this method with a\r\n* Bot being the bot creating the conversation\r\n* IsGroup set to true if this is not a direct message (default is false)\r\n* Array containing the members to include in the conversation\r\n\r\nThe return value is a ResourceResponse which contains a conversation id which is suitable for use\r\nin the message payload and REST API uris.\r\n\r\nMost channels only support the semantics of bots initiating a direct message conversation.  An example of how to do that would be:\r\n\r\n```\r\nvar resource = await connector.conversations.createConversation({\r\n    bot,\r\n    members: [{ id: 'user1' }]\r\n});\r\nawait connector.conversations.sendToConversation(resource.Id, ... );```",
```

Comment:
You should sync with Axel or Eric on this or have one of them approve the PR, Axel is leading the work around improving our Swagger file management.

See https://github.com/microsoft/botframework-sdk/issues/5945 and https://github.com/microsoft/botframework-sdk/pull/5946

Reply:
Resolved via https://github.com/microsoft/botbuilder-js/pull/2476#pullrequestreview-442555713